### ISSUE TITLE
still active?

### ISSUE THREAD

**Issue by ttztony** [shivamsharma00/sups_story_teller#1]
are you still working on this?

**Comment by shivamsharma00** [comment]
Yes, I am a little slow on this project. Till now I have collected the data for story teller.

**Comment by ttztony** [comment]
> Yes, I am a little slow on this project. Till now I have collected the data for story teller.

Interesting project. You on discord? Let's have a chat.

**Comment by shivamsharma00** [comment]
Yeah sure, I got discord. Ping me on galeetch.
